couchbase/admin/searchindex/v1/searchindex.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+option go_package = "github.com/couchbase/goprotostellar/genproto/admin_searchindex_v1;admin_searchindex_v1";
+option ruby_package = "Couchbase::Protostellar::Generated::Admin::SearchIndex::V1";
+option php_namespace = "Couchbase\\Protostellar\\Generated\\Admin\\SearchIndex\\V1";
+option java_package = "com.couchbase.client.protostellar.admin.searchindex.v1";
+option java_multiple_files = true;
+
+package couchbase.admin.searchindex.v1;
+
+service SearchIndexAdminService {
+    rpc GetSearchIndex(GetSearchIndexRequest) returns (GetSearchIndexResponse) {}
+    rpc ListSearchIndexes(ListSearchIndexesRequest) returns (ListSearchIndexesResponse) {}
```

Comment:
OOI why List rather than SDK's GetAll?

Reply:
I believe that people have typically used "List" rather than "GetAll" in protobuffers.